api/python/provisioner/inputs.py

```diff
@@ -1113,23 +1113,26 @@ def pillar_value(self):
             return res
         elif self.is_remote():
             return {
-                f'os': {
-                    'source': (f'{self.source}/os'),
+                f'{config.OS_ISO_DIR}': {
+                    'source': f'{self.source}/os/{config.OS_ISO_DIR}',
                     'is_repo': True,
                     'enabled': self.enabled
                 },
-                f'cortx_iso': {
-                    'source': (f'{self.source}/cortx_iso'),
+                f'{config.CORTX_ISO_DIR}': {
+                    'source': f'{self.source}/cortx_iso/'
```

Comment:
to ensure that concatenation will take please please add parenthesis here and in similar places

Reply:
Fixed